activation/nipost_test.go

```diff
@@ -180,6 +180,9 @@ func TestPostSetup(t *testing.T) {
 }
 
 func TestNIPostBuilderWithClients(t *testing.T) {
+	if os.Getenv("GOOS") == "windows" {
```

Comment:
plz create named constants for these OSs and env variables so every test in all packages can use them.

Reply:
I'm not certain it gives any benefit. IMHO, it would obfuscate the code. Named constants are useful if the constant might change in the future. Then such a change is easier, a one-liner. However, here we know for sure these constants will never change.

Anyway, where do you suggest putting them?

Reply:
sure these constants will never change, but typos in the new code can happen.

for example
`if os.Getenv("GOOS") != "wondows"` {
    // skip
}
then one will never notice because it never run on any OS.
common/utils?